frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { checkErrors } from '../../support';
+import { timestamp } from '../../views/timestamp';
+import { shouldMatchRegex } from '../../views/common';
+
+describe('Localization', () => {
+  before(() => {
+    cy.login();
```

Comment:
I guess you don't have to create a test project?

Reply:
Do I need to? If I need to, I'll add it back in.

Reply:
That what I was questioning :-)  I guess not if all the tests i18n tests work.  I believe we only create projects if we CRUD resources, so when we delete the test project all artifacts are also cleanned up.  So, if i18n doesn't CRUD anything, I think we're good